spec/compiler/semantic/block_spec.cr

```diff
@@ -1075,7 +1075,7 @@ describe "Block inference" do
 
       bar
       ),
-      "can't break from captured block, use `next`, however `next` will have behave differently when used in a block or a loop inside of the captured block."
+      "can't break from captured block, try using `next` to exit the block."
```

Comment:
I think stating that `next` exits the block is misleading exactly in the way we're trying to clarify things.